pytorch_toolkit/object_detection/tests/export_tests_person_detection.py

```diff
@@ -48,3 +48,15 @@ class PersonDetection0202TestCase(
             True)
 ):
     """ Test case for person-detection-0202 model export. """
+
+
+class PersonDetection0203TestCase(
+        create_export_test_case(
+            'person-detection',
+            'person-detection-0203',
+            os.path.dirname(__file__) + '/../../../data/airport/annotation_example_train.json',
```

Comment:
try to use proper annotation here and above 'annotation_person_train.json', I am fixing it in my PR. Looks like CI job is hanging on loading http://213.221.44.203/job/ote/job/precommit/job/pt_object_detection/505/consoleFull. I kill it.